api/core/workflow/nodes/agent/agent_node.py

```diff
@@ -151,7 +152,10 @@ def _generate_agent_parameters(
             if (parameter_value.startswith("{") and parameter_value.endswith("}")) or (
                 parameter_value.startswith("[") and parameter_value.endswith("]")
             ):
-                value = literal_eval(parameter_value)  # transform string to python object
+                try:
+                    value = literal_eval(parameter_value)
+                except:
```

Comment:
Please do not use [bare except](https://www.flake8rules.com/rules/E722.html). Catch the following errors instead ([ref](https://docs.python.org/3/library/ast.html#ast.literal_eval)):

```
except (ValueError, TypeError, SyntaxError)
```